test/syscalls/linux/kill.cc

```diff
@@ -103,6 +103,118 @@ TEST(KillTest, CanKillAllPIDs) {
   EXPECT_EQ(0, WEXITSTATUS(status));
 }
 
+class KillSpecificReaperTest
+    : public ::testing::TestWithParam<
+          std::tuple<std::function<int(void *)>>> {
+ public:
+  static int ReaperEntryPoint(void *args) {
+    return std::get<0>(GetParam())(args);
```

Comment:
We don't need to use a tuple.  This can just be:

```
return GetParam()(args);
```

We also can just get rid of the `ReaperEntryPoint` method, and directly call `GetParam()` in the test body.  But the helper is OK too.  I can leave that up to you to decide.

Reply:
Thanks, I have changed the code to simple and clear way, which will directly call `GetParam()` in test body.